### DISCUSSION TITLE
Can we add home info in every pages?

### DISCUSSION THREAD

**Discussion by seanmrnda** [adityatelange/hugo-PaperMod#933] [Ideas]
```
homeInfoParams:
        Title: "PaperMod"
        Content: " "
```
### Question:
Is it possible to add a Home Info in every pages?
### Idea:
It would be a great feature if we can add home info in any page we want. Like for example, adding it in ***Posts***.

**Comment by kaitoukid-1412** [comment]
delete code `(eq $paginator.PageNumber 1)`  in `layouts/_default/list.html` line [47](https://github.com/adityatelange/hugo-PaperMod/blob/master/layouts/_default/list.html#L47)
